src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,10 @@
+package duke;
+
+/**
+ * Custom exception to be thrown when there is an error parsing user input, or if task list fails to load or save.
+ */
+public class DukeException extends Exception {
```

Comment:
It may be good to have different classes of exceptions for more Duke-specific exceptions